src/lib/throttle.js

```diff
@@ -132,3 +136,55 @@ export function getReturnToken() : ?string {
         return token;
     }
 }
+
+type CustomThrottle = Throttle & {
+    isActive : () => boolean
+};
+
+export function buildFundingLogoThrottle(props : Object) : CustomThrottle {
+
+    let emptyThrottle = {
+        isActive:     () => false,
+        isEnabled:    () => false,
+        isDisabled:   () => true,
+        getTreatment: () => '',
+        log:          () => emptyThrottle,
+        logStart:     () => emptyThrottle,
+        logComplete:  () => emptyThrottle
+    };
+
+    let { layout, label, locale, browserLocale } = props;
+
+    if (browserLocale.country !== COUNTRY.US) {
+        return emptyThrottle;
+    }
+
+    let localeString = locale;
+
+    if (typeof locale === 'object') {
+        localeString = `${ locale.lang }_${ locale.country }`;
+    }
+
+    if (localeString !== 'en_US') {
+        return emptyThrottle;
+    }
+
+    if (label !== 'checkout' && label !== 'paypal' && label !== 'pay' && label !== 'buynow') {
```

Comment:
Can you use the `BUTTON_LABEL` const here?

Reply:
Yes I can.